apis/logging/v1/cluster_log_forwarder_types.go

```diff
@@ -173,6 +173,10 @@ type OutputSpec struct {
 	//   `passphrase`: Passphrase to decode an encoded TLS private key. Requires tls.key.
 	//   `ca-bundle.crt`: Custom CA to validate certificates.
 	//
+	//   `tls.insecure`: If this key is present (with any value) accept server certificates that are self-signed or have an incorrect hostname.
```

Comment:
Do we want to propagate this to the formal API?  If it is our tests in CI we have ways to manipulate the config string

Reply:
Yes - this is intended for customers more than us. Customers forwarding off-cluster often need to test forwarding in a lab before they deploy.  In a lab setting, the target has self-signed or hostname-mismatched certs, hence need for this option. This came up with Verizon & their test Kafka installation, I think it will be common. 
I'm hoping the word "insecure" is enough of a tip that this isn't intended for production.